presto-native-execution/src/test/java/com/facebook/presto/hive/HiveExternalWorkerQueryRunner.java

```diff
@@ -154,7 +154,7 @@ public static QueryRunner createNativeQueryRunner(String baseDataDir, String pre
                         Files.write(tempDirectoryPath.resolve("node.properties"),
                                 format("node.id=%s%n" +
                                         "node.ip=127.0.0.1%n" +
-                                        "node.environment=testing%n" +
+                                        "node.environment=test%n" +
```

Comment:
Would you double check that with this change everything works? I remember "testing" being hard-coded somewhere in the coordinator, so wondering if more changes are needed.